dev-requirements.txt

```diff
@@ -3,18 +3,18 @@
 # running test suites, and mocking Python Objects for the Scalyr Agent.
 asyncio==3.4.3
 coverage==4.4.1
-mock==2.0.0
+mock==0.8.0
```

Comment:
Downgraded mock library to work with lower versions of python. Verified it still works with 2.7